src/assets/config/changelog.json

```diff
@@ -1 +1,125 @@
-{}
+{
+  "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/release/v2.18/CHANGELOG.md#kubermatic-218",
+  "entries": [
+    {
+      "category": "action-required",
+      "description": "Helm 2 is not supported anymore, please use Helm 3 instead. It might still be possible to install KKP using Helm 2, but it's neither supported nor recommended."
+    },
+    {
+      "category": "action-required",
+      "description": "The new backup mechanism replaces the old mechanism and is not backwards compatible. Furthermore the admin of the platform needs to enable the backup mechanism as described in the documentation.",
+      "links": [
+        {
+          "url": "https://docs.kubermatic.com/kubermatic/v2.18/cheat_sheets/etcd/backup-and-restore/",
+          "caption": "Documentation"
+        }
+      ]
+    },
+    {
+      "category": "added",
+      "description": "User Cluster Monitoring, Logging and Alerting",
+      "links": [
+        {
+          "url": "https://docs.kubermatic.com/kubermatic/v2.18/guides/monitoring_logging_alerting/user_cluster/",
+          "caption": "MLA"
+        }
+      ]
+    },
+    {
+      "category": "added",
+      "description": "Cluster Templates for Deploying Optimal Clusters Instantly",
```

Comment:
It's a bit inconsistent because some of the descriptions are using title case but some are not. I guess this is a copy from original changelog so we should raise this point next time.